cmake/winml_unittests.cmake

```diff
@@ -36,7 +36,7 @@ function(add_winml_test)
     list(REMOVE_DUPLICATES _UT_DEPENDS)
   endif()
 
-  add_executable(${_UT_TARGET} ${_UT_SOURCES})
+  add_executable(${_UT_TARGET} ${_UT_SOURCES} ${WINML_TEST_SRC_DIR}/common/googletest/main.cpp)
```

Comment:
We weren't building the same file more than once. Adding `googletest/main.cpp` to each test target will build it multiple times, can you move it into one of the reusable static libraries, or create one for it?